src/nile/cli.py

```diff
@@ -145,11 +148,25 @@ async def declare(
 
 @cli.command()
 @click.argument("signer", nargs=1)
+@click.option("--salt", nargs=1)
+@click.option("--max_fee", nargs=1)
 @network_option
 @watch_option
-async def setup(signer, network, watch_mode):
+async def setup(signer, network, salt, max_fee, watch_mode):
     """Set up an Account contract."""
-    await Account(signer, network, watch_mode=watch_mode)
+    await Account(signer, network, salt, max_fee, watch_mode=watch_mode)
+
+
+@cli.command()
+@click.argument("signer", nargs=1)
+@click.option("--salt", nargs=1, default=None)
+async def counterfactual_address(signer, salt):
+    """Precompute the address of an Account contract."""
+    _signer = Signer(normalize_number(os.environ[signer]))
+    address = hex_address(
+        get_counterfactual_address(salt, calldata=[_signer.public_key])
```

Comment:
To follow up the above comment, maybe `compute_address` instead of `get_counterfactual_address`?